wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/SchemaHandler.kt

```diff
@@ -0,0 +1,97 @@
+package com.squareup.wire.schema
+
+import com.squareup.wire.WireLogger
+import com.squareup.wire.internal.Serializable
+import okio.FileSystem
+import okio.Path
+
+/**
+ * A [SchemaHandler] [handle]s [Schema]!
+ * Implementations of this interface must have a no-arguments public constructor. TODO(Benoit) Is it still true?
```

Comment:
For your TODO, not still true. The _Factory_ must have a no-arguments public constructor if it is named in a Gradle task